### ISSUE TITLE
Rex Ryan in shambles

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_English-GUM#54]
`toes` is lemmatized to `to` and not `toe`

```
# sent_id = GUM_conversation_lambada-49
41      and     and     CCONJ   CC      _       42      cc      42:cc   Discourse=joint-sequence_m:97->91:1
42      goes    go      VERB    VBZ     Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin   15      conj    15:conj:and     _
43      along   along   ADP     IN      _       45      case    45:case _
44      his     his     PRON    PRP$    Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs  45      nmod:poss       45:nmod:poss    Entity=(60-object-new-2-sgl(54-person-giv:act-1-ana)
45      toes    to      NOUN    NNS     Number=Plur     42      obl     42:obl:along    Entity=60)|SpaceAfter=No
```

**Comment by amir-zeldes** [comment]
Will fix, thanks for reporting!

**Comment by AngledLuffa** [comment]
fixed in 2.11
